Docs: Adding a step to verify Authorized Applications (#2624)
codenvy/che-docs: modules/administration-guide/partials/proc_setting-up-the-gitlab-authorized-application.adoc

$ echo -n '__<gitlab_client_secret>__' | base64
----

> . Verify that the Application is Authorized, Click your avatar and go to *Edit Profile* → *Applications* → *Authorized Applications*

Reviewer: ```suggestion
. Verify that the application is authorized by clicking on your avatar and selecting *Edit Profile* → *Applications* → *Authorized Applications*
```
